src/components/my-view1.js

```diff
@@ -22,13 +22,13 @@ class MyView1 extends PageViewElement {
         <p>It doesn't do anything other than display some static text.</p>
       </section>
       <section>
-        <h2>Bacon ipsum</h2>
-        <p>Bacon ipsum dolor amet swine picanha doner pig. Buffalo pastrami ball tip brisket biltong spare ribs, flank pork chop strip steak pork loin venison pig porchetta chuck. Turducken biltong porchetta spare ribs tail. Shoulder jowl shankle kevin, ground round burgdoggen short loin turkey alcatra pork loin buffalo prosciutto beef ribs sirloin. Filet mignon alcatra tail tongue. Leberkas kevin jowl meatloaf.</p>
-        <p>Capicola ham short ribs, strip steak turkey ham hock picanha. Biltong fatback boudin pork chop buffalo venison. Ground round buffalo picanha leberkas doner ribeye tongue kielbasa meatball andouille bacon. Chicken spare ribs corned beef ball tip tri-tip frankfurter chuck.</p>
+        <h2>Cheese ipsum</h2>
+        <p>Squirty cheese roquefort goat. Port-salut monterey jack cheesy grin gouda airedale when the cheese comes out everybody's happy fondue edam. Cauliflower cheese lancashire rubber cheese pepper jack cheese and wine who moved my cheese cheese and biscuits emmental. Goat cheese slices cheese strings cut the cheese port-salut hard cheese gouda cheese on toast. Cheesy feet paneer lancashire cheese triangles brie hard cheese ricotta blue castello. Taleggio cut the cheese blue castello roquefort cut the cheese fondue roquefort caerphilly. Stinking bishop cow brie fondue cheese and biscuits croque monsieur danish fontina cottage cheese. Pepper jack the big cheese squirty cheese pecorino dolcelatte dolcelatte fromage frais brie. Cheese on toast chalk and cheese manchego paneer lancashire lancashire caerphilly.</p>
```

Comment:
Can we just change "squirty cheese" to cheese (here and below) bc it sounds weird. 🙏